app/models/conference.rb

```diff
@@ -838,7 +838,7 @@ def rooms_set?
   # * +True+ -> If conference has a venue object.
   # * +False+ -> IF conference has no venue object.
   def venue_set?
-    !!venue
+    !venue.nil?
```

Comment:
You can replace all of these `!something.nil?` with `something.present?` , which, personally, I find much more readable.